eth_tester/backends/__init__.py

```diff
@@ -10,36 +14,52 @@
     EELSBackend,
     eels_is_available,
 )
-from .mock import (
-    MockBackend,
-)
 from .pyevm import (
     PyEVMBackend,
+    is_supported_eels_version_available,
```

Comment:
why are we importing `is_supported_eels_version` from pyevm? This is what `eels_is_available` is for, I believe, no?